File: BACKEND/Application/Groups/Responses/GroupMembersResponse.cs
using Application.Users.Responses;

namespace Application.Groups.Responses
{
    public class GroupMembersResponse
    {
        public List<UserBaseResponse> Members { get; set; } = [];
        public int MaxModeratorNumber { get; set; }
        public int CurrentModeratorNumber { get; set; }
        public int MaxMemeberNumber { get; set; }
        public int CurrentMemberNumber { get; set; }
    }
}

File: FRONTEND/backgammon-frontend/src/app/features/groups/models/api/responses/group-members.response.ts
import { UserBaseResponse } from '../../../../user/models/api/responses/user-base.response';

export interface GroupMembersResponse {
  members: UserBaseResponse[];

  maxModeratorNumber: number;

  currentModeratorNumber: number;

  maxMemeberNumber: number;

  currentMemberNumber: number;
}
